tests/qchem/of_tests/test_convert.py

```diff
@@ -987,15 +999,134 @@ def test_excited_configurations_error(excitation):
         _ = qchem.convert._excited_configurations(2, 4, excitation)
 
 
-def test_fail_import_pyscf(monkeypatch):
-    """Test if an ImportError is raised when pyscf is requested but not installed."""
+@pytest.mark.parametrize(
+    ("molecule", "basis", "symm", "tol", "wf_ref"),
+    [
+        (
+            [["H", (0, 0, 0)], ["H", (0, 0, 0.71)]],
+            "sto6g",
+            "d2h",
+            1e-1,
+            {(1, 1): -0.9942969785398778, (2, 2): 0.10664669927602179},
+        ),
+        (
+            [["H", (0, 0, 0)], ["H", (0, 0, 0.71)]],
+            "cc-pvdz",
+            "d2h",
+            4e-2,
+            {
+                (1, 1): 0.9919704795977625,
+                (2, 2): -0.048530356564386895,
+                (2, 8): 0.044523330850078625,
+                (4, 4): -0.050035945684911876,
+                (8, 2): 0.04452333085007864,
+                (8, 8): -0.052262303220437775,
+                (16, 16): -0.040475973747662694,
+                (32, 32): -0.040475973747662694,
+            },
+        ),
+        (
+            [["Be", (0, 0, 0)]],
+            "sto6g",
+            "d2h",
+            1e-3,
+            {
+                (3, 3): 0.9446343496981953,
+                (6, 5): 0.003359774446779245,
+                (10, 9): 0.003359774446779244,
+                (18, 17): 0.003359774446779245,
+                (5, 6): 0.003359774446779244,
+                (5, 5): -0.18938190575578503,
+                (9, 10): 0.003359774446779243,
+                (9, 9): -0.18938190575578523,
+                (17, 18): 0.003359774446779244,
+                (17, 17): -0.18938190575578503,
+            },
+        ),
+    ],
```

Comment:
These are also almost identical (except for Be) between UCISD and RCISD tests, maybe you can add the missing case and define these at the beginning of the file as well.

Reply:
Done.